com/yadro/HWManager/StorageManager.interface.yaml

```diff
@@ -0,0 +1,62 @@
+description: >
+    The interface to control Yadro storage subsystem.
+
+methods:
+    - name: FindDrive
+      description: >
+          Return port name where the rive insrted.
+      parameters:
+          - name: DriveSN
+            type: string
+            description: >
+                Serial number of the drive, to looking for
+      returns:
+          - name: Result
+            type: string
+            description: >
+                Port name
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
+          - xyz.openbmc_project.Common.Error.InternalFailure
+          - xyz.openbmc_project.Common.Error.ResourceNotFound
+
+    - name: SetDriveLocationLED
+      description: >
+          Turn On/Off drive location LED.
+      parameters:
+          - name: DriveSN
+            type: string
+            description: >
+                Serial number of the drive, corresponding to required LED
```

Comment:
"Serial number of the drive for which the LED is to be set". No comma is required before restrictive clauses in English.

The same applies to the similar text in other descriptions below.